src/main/java/com/birbit/jsonapi/JsonApiMetaDeserializer.java

```diff
@@ -36,6 +38,14 @@ public JsonApiMeta deserialize(JsonElement json, Type typeOfT, JsonDeserializati
             JsonElement value = entry.getValue();
             if (value.isJsonPrimitive()) {
                 result.put(entry.getKey(), entry.getValue().getAsString());
+            } else if (value.isJsonArray()) {
+                List<String> allValues = new ArrayList<>();
+                for (JsonElement element : entry.getValue().getAsJsonArray()) {
```

Comment:
Just tiny bit - we can change entry.getValue() with value as we already have it. This can also be replaced in the if statement

Reply:
Updated.